CoastalResilienceNetwork/regional-planning
# New design style fixes

## Overview

Fixes a few UI issues so that the plugin renders correctly in the new design.

### Demo

![image](https://cloud.githubusercontent.com/assets/1042475/20061913/c5a9d1bc-a4cf-11e6-8071-7be8e73ecd63.png)
*Tab styling*


![image](https://cloud.githubusercontent.com/assets/1042475/20062675/8a40373a-a4d2-11e6-9f81-97cf43afdd8b.png)
*Icons are fixed*


![image](https://cloud.githubusercontent.com/assets/1042475/20061989/07c36c3e-a4d0-11e6-88e5-673a954c0696.png)
*Layer tool Icons are back*

**Testing**
- Check out https://github.com/CoastalResilienceNetwork/GeositeFramework/pull/727.
- Verify the correct icons are used in the layer selector.
- Verify the layer tool icons are shown.
- Verify the the tabs are styled and work.
- Verify there are no requests for components of the Foundation framework.

**Notes**
- One remaining issue is to fix the layer info box. The current styling does not work with the new design markup. I made several simple attempts to fix it, but I couldn't figure out something that worked well. https://github.com/CoastalResilienceNetwork/regional-planning/issues/63

Connects to #59
